client/jetpack-cloud/sections/partner-portal/primary/payment-method-add/index.tsx

```diff
@@ -25,11 +111,80 @@ export default function PaymentMethodAdd(): ReactElement {
 				<CardHeading size={ 36 }>{ translate( 'Payment Method' ) }</CardHeading>
 			</div>
 
-			<Card className="payment-method-add__body">
-				<div className="payment-method-add__body-left"></div>
+			<CheckoutProvider
+				onPaymentComplete={ () =>
+					onPaymentSelectComplete( {
+						successCallback: () => page( '/partner-portal/payment-method/' ),
+						translate,
+						showSuccessMessage,
+					} )
+				}
+				showErrorMessage={ showErrorMessage }
```

Comment:
Please be aware that the API for `CheckoutProvider` has changed in https://github.com/Automattic/wp-calypso/pull/55809 regarding messages.